test/unit/helpers/dom/EnrollChoicesForm.js

```diff
@@ -26,16 +26,40 @@ define(['okta', './Form'], function (Okta, Form) {
       return this.factorDescription(factorName).find('p').trimmedText();
     },
 
-    factorButton: function (factorName) {
-      return this.factorRow(factorName).find('.button');
+    factorButton: function (factorName, factorsRow) {
```

Comment:
would it be better to have 2 functions?
```
factorButton: function (factorName) {
...
}

rowButton: function (row) {
...
}
```
Same question for the other functions.

Reply:
This function gets the factorButton of a particular row. Specifically the setup button for optional factors. 
With multiple enrollments, we can now have multiple rows with the same name.  So i'm providing the option to pass in the row itself, to get the button corresponding to that row. 
If i were to break it into two functions, it may seem like those two functions are returning different types of buttons. I think we should keep it as one function. What do you think?

Reply:
ok... I didn't feel strong one way or the other... I was just trying to avoid passing `null` as a first parameter, but what you said makes sense, so we can leave it as is 👍